lib/mongo_mapper/plugins.rb

```diff
@@ -14,9 +16,18 @@ def plugin(mod)
       plugins << mod
     end
 
+    def direct_descendants
+      Plugins.direct_descendants_hash[self]
+    end
+
```

Comment:
ActiveSupport::DescendantsTracker#direct_descendants doesn't work with Modules with Ruby 3.0 and ActiveSupport 7.0. So let's implement it ourselves.